src/sqlite3/result_set.cr

```diff
@@ -44,25 +44,57 @@ class SQLite3::ResultSet < DB::ResultSet
         raise Exception.new(sqlite3_statement.sqlite3_connection)
       end
     @column_index += 1
-    value
+    value.as(Float64 | Int64 | Slice(UInt8) | String | Time | Nil)
```

Comment:
I see, I didn't realized 4247 and 4176 were the same.

`value.as(DB::Any)` will do the trick for now.

crystal-db 🔨  & 🔧  day.